src/libDB/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+find_package(libmongocxx REQUIRED)
+find_package(libbsoncxx REQUIRED)
+include_directories(${LIBMONGOCXX_INCLUDE_DIR})
+include_directories(${LIBBSONCXX_INCLUDE_DIR})
+include_directories("/usr/local/include/mongocxx/v_noabi")
+include_directories("/usr/local/include/bsoncxx/v_noabi")
+include_directories("/usr/local/include/libmongoc-1.0")
+include_directories("/usr/local/include/libbson-1.0")
+include_directories("/usr/local/lib")
```

Comment:
Need some clean-ups on these `include_directories` commands. Try to keep it minimum.

Reply:
Also better to make a decision on which file to include these commands. It doesn't look good to me that both top-level CMakeLists.txt and `src/libDB/CMakeLists.txt` have these lines.